src/pudl/package_data/notebooks/how_to_use_pudl_out.ipynb

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
<p><code>help(pudl_out)</code> provides a list of all the public methods, as well as their (relatively short) docstrings, which is very informative as to what's available, and seems more intuitive (and also works on other Python objects)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/844/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>hm, but the output of that is more than 400 lines. which is too overwhelming, imo. maybe we can include this as a commented out option.</p>

Reply:
I know it's a lot of output, but the other way is an inscrutable incantation that nobody will ever remember or be able to re-use on their own in another context, whereas `help(object)` is simple and memorable and will work in lots of contexts. We might also want to mention tab-completion in Jupyter Notebooks, and shift-tab to see a list of possible completions.

Reply:
<p>but it is over  400 lines. it is too overwhelming. I added it right now with a note but commented it out... because it is 400 lines.</p>